pkg/interactive-installer/build.yml

```diff
@@ -0,0 +1,48 @@
+org: lfedge
+image: eve-interactive-installer
+network: yes
```

Comment:
This should be `network: no` (or just don't write the line, as it is the default).